src/Symfony/Component/Validator/ConstraintValidatorFactory.php

```diff
@@ -24,11 +24,8 @@
  */
 class ConstraintValidatorFactory implements ConstraintValidatorFactoryInterface
 {
-    protected array $validators;
-
-    public function __construct(array $validators = [])
+    public function __construct(protected array $validators = [])
     {
```

Comment:
When using CPP, we always use multiline parameter lists.

```suggestion
    public function __construct(
        protected array $validators = []
    ) {
```

Also applies to other occurrences in this PR.